cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,286 @@
+
+        self.assertNotIn(movie_serializer_japon.data, movie_response_apalon.data)
+        self.assertNotIn(movie_serializer_apalon.data, movie_response_vinston.data)
+
+        self.assertEqual(movie_response_lolipops.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_response_lolipops.data, [])
+
+    def test_filter_movie_by_actors(self):
+        movie_responce_Illia = self.client.get(
+            MOVIE_LIST_URL, {"actors": f"{self.actor_Illia.id}"}
+        )
+        movie_responce_Jonh = self.client.get(
+            MOVIE_LIST_URL, {"actors": f"{self.actor_John.id}"}
+        )
+        movie_responce_qeuty = self.client.get(
+            MOVIE_LIST_URL, {"actors": f"918"}
+        )
+
+        movie_serializer_Jonh = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_Illia = MovieListSerializer(self.movie_vinston, many=False)
+
+
+        self.assertEqual(
+            movie_serializer_Illia.data["actors"],
+            movie_responce_Illia.data[0]["actors"],
+        )
+        self.assertEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Jonh.data[0]["actors"]
+        )
+
+        self.assertNotEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Illia.data[0]["actors"]
+        )
+
+        self.assertEqual(movie_responce_qeuty.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_responce_qeuty.data, [])
+
+    def test_filter_movie_by_genres(self):
+        movie_response_horror = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_horror.id}"}
+        )
+        movie_response_drama = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_drama.id}"}
+        )
+        movie_response_something = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"89982"}
+        )
+
+        movie_serializer_horror = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_drama = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertEqual(
+            movie_serializer_horror.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_drama.data[0]["genres"],
+        )
+        self.assertNotEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(movie_response_something.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_response_something.data, [])
+
+    def test_filter_movie_session_by_date(self):
+        movie_session_response_2023 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": f"{self.movie_session_2023.show_time.year}-"
+                f"{self.movie_session_2023.show_time.month}-"
+                f"{self.movie_session_2023.show_time.day}"
```

Comment:
You're manually constructing the date string here. Consider using the strftime method of the datetime object to format the date string. This would make the code cleaner and easier to read.